react/src/lib/Slider/index.js

```diff
@@ -155,10 +155,24 @@ class Slider extends React.Component {
     };
 
     return (
-      <div className={
-        `md-slider ${className}` +
-        `${(disabled && ` md-slider--disabled`) || ''}` +
-        `${(className && ` ${className}`) || ''}`
```

Comment:
Also this is a bit redundant because className is already added 2 lines up. I suggest we remove that and stay with this conditional line.